perception/radar_tracks_msgs_converter/README.md

```diff
@@ -18,12 +18,7 @@ This package convert from [radar_msgs/msg/RadarTracks](https://github.com/ros-pe
 
 ### Parameters
 
-- `update_rate_hz` (double): The update rate [hz].
-  - Default parameter is 20.0
-- `new_frame_id` (string): The header frame of output topic.
-  - Default parameter is "base_link"
-- `use_twist_compensation` (bool): If the parameter is true, then the twist of output objects' topic is compensated by ego vehicle motion.
-  - Default parameter is "false"
+{{ json_to_markdown("perception/radar_tracks_msgs_converter/schema/radar_tracks_msgs_converter.schema.json") }}
```

Comment:
[MUST]
This change is very inconvenience because we cannot see parameters description at github.
I think the parameter description should be written by markdown like [this PR](https://github.com/autowarefoundation/autoware.universe/pull/4871/files).

Reply:
@scepter914 This is an implementation of the goal described in https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/#workflow. Could you consider using the rendered version of the documentation instead of the github pages? For example: https://autowarefoundation.github.io/autoware.universe/latest/perception/lidar_apollo_segmentation_tvm_nodes/#parameters

Reply:
Why we use only the rendered version of documents?
I think it is convenience both github README and the rendered version.

In addition, as the reaction in PR of https://github.com/autowarefoundation/autoware.universe/pull/5241 is neglected for now, I cannot recognize the maintenance ability for using the json schema for parameter tables.
As code owner, I cannot accept the change that cause inconvenience for maintenance, and it is not allowed to delete the markdown parameter table unless there is enough maintenance ability.